product/src/it/java/com/yas/product/controller/ProductControllerIT.java

```diff
@@ -517,6 +517,19 @@ void test_subtractProductQuantity_shouldSuccess() {
 
     }
 
+    @Test
+    void test_getProductCheckoutList_shouldReturnProductList() {
+        initProductData();
```

Comment:
No need to initialize data as it has been done in BeforeEach method (line 188)

Reply:
@NhatTranMinh15 I've already fixed it, please take a look, thank you!